app/_data/docs_nav_ee_0.35-x.yml

```diff
@@ -103,8 +149,18 @@
     url: /admin-api/#target-object
   - text: Workspace Object routes
     url: /admin-api/workspaces/reference
+    items:
+      - text: Workspaces Reference
+        url: /admin-api/workspaces/reference
+      - text: Workspace Examples
+        url: /admin-api/workspaces/examples
   - text: RBAC Object routes
     url: /admin-api/rbac/reference
+    items:
+      - text: RBAC Reference
+        url: /admin-api/rbac/reference
+      - text: RBAC Examples
+        url: /admin-api/rbac/examples
   - text: Admins
```

Comment:
I think these were left out because they need updating (the easy fix in the interest of time? I don't know the full history), but it does make sense to bring them in since people still use the topics. An FYI as well: by necessity, this is probably a temp location in the nav, as we want to automate the Admin API guide. But we can revisit that then...

Reply:
I'm not sure if they're outdated, possibly (I didn't read them end-to-end). But they're extremely important - e.g. the `Workspace Examples` page is the only place that explains conflict resolution between workspaces, and the `RBAC Examples` page is the only place that shows actual use of the `Kong-Admin-Token` to authenticate to the Admin API.

I stumbled on both of these "hidden pages" more or less by accident when I was searching for these topics.

Reply:
I used the RBAC examples page to figure out RBAC for myself and did run into some places where it didn't work as described, so it's definitely something that needs another look. However, hiding these pages doesn't solve anything, and at least exposing them first is a step forward - I want to work on incremental improvements here going forward.